test/functional/fixtures/regression/gh-993/test.js

```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-993)', function () {
+    it('t.pressKey should work properly on Angular2 page', function () {
```

Comment:
It should be something more specified, like
`key events should have 'key' or 'keyIdentifier' properties when t.pressKey('enter') performed`